packages/atomic/playwright-utils/cache.ts

```diff
@@ -0,0 +1,40 @@
+import {APIResponse, Request, Route} from '@playwright/test';
+import {createHash} from 'crypto';
```

Comment:
I remember you talked about browser compat for crypto.
fwiw: `globalThis.crypto.subtle.digest` would probably do a similar trick as createHash, for both NodeJS & the browser